pkgs/development/tools/misc/fujprog/default.nix

```diff
@@ -25,13 +26,13 @@ stdenv.mkDerivation rec {
   buildInputs = [
     libftdi1
     libusb-compat-0_1
-  ];
+  ] ++ stdenv.lib.optionals stdenv.isDarwin IOKit;
 
   meta = with stdenv.lib; {
-    description = "JTAG programmer for the ULX3S and ULX2S open hardware FPGA development boards.";
+    description = "JTAG programmer for the ULX3S and ULX2S open hardware FPGA development boards";
     homepage = "https://github.com/kost/fujprog";
     license = licenses.bsd2;
     maintainers = with maintainers; [ trepetti ];
-    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
please do not remove `platforms`
```
platforms = platforms.unix;
```

Reply:
https://github.com/NixOS/nixpkgs/pull/96519#discussion_r479613615

The `CMakeLists.txt` covers MacOS, Linux and Windows. I have not tested Windows explicitly, but should it be left to `platforms.all` (i.e., default) instead of restricted in this case?

Reply:
If the program supports Windows, I don't think it is a bid idea to leave this at `platforms.all`.

Reply:
Will leave on all for now pending any Windows issues that crop up.

Reply:
If you remove `platforms`, hydra won't build this package. [From the docs](https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes):
>Hydra builds packages according to the platform specified. If no platform is specified, the package does not have prebuilt binaries

Reply:
Back to `platforms.all` now. Thanks!